tests/test_basic_twoword_classifier.py

```diff
@@ -0,0 +1,28 @@
+import numpy as np
+import torch
+from scripts import BasicTwoWordClassfier
+import unittest
+
+
+class BasicTwoWordClassifierTest(unittest.TestCase):
+    """
+    this class tests the BasicTwoWordClassifier
+    This test suite can be ran with:
+        python -m unittest -q tests.BasicTwoWordClassifierTest
+    """
+
+    def setUp(self):
+        self.word1 = torch.from_numpy(np.array([[1, 0, 0]], dtype='float32'))
+        self.word2 = torch.from_numpy(np.array([[0, 1, 0]], dtype='float32'))
+        self.inputdim = 6
```

Comment:
for consistency: let's stick to pep conventions and use input_dim , hidden_dim ... just like you did in the classifier.